src/llv8.cc

```diff
@@ -1838,10 +1838,10 @@ v8::Value JSObject::GetArrayElement(int64_t pos, Error& err) {
 }
 
 std::string JSArray::Inspect(InspectOptions* options, Error& err) {
-  Smi length = Length(err);
+  int64_t length = GetArrayLength(err);
```

Comment:
The original code would call `NameDictionary::Length()` instead so the length displayed would count an additional entry (`length`).